cohort-engine-api/src/main/java/com/ibm/cohort/engine/api/service/FHIRRestUtils.java

```diff
@@ -153,7 +153,7 @@ public static IGenericClient getFHIRClient(String fhirEndpoint, String bearerTok
 		RestFhirMeasureResolutionProvider msp = new RestFhirMeasureResolutionProvider(measureClient);
 		Measure measure = msp.resolveMeasureByIdentifier(identifier, measureVersion);
 
```

Comment:
Think we need a null check here on the measure as well, no?

Reply:
Yep, good call.
I forgot there were two codepaths for getting measures.